zellij-server/src/plugins/mod.rs

```diff
@@ -754,14 +763,24 @@ pub(crate) fn plugin_thread_main(
                     pipe_messages.push((
                         Some(plugin_id),
                         Some(client_id),
-                        PipeMessage::new(PipeSource::Keybind, name, &payload, &args, is_private),
+                        PipeMessage::new(
+                            PipeSource::Keybind,
+                            Some(client_id),
+                            pane_id.map(Into::into),
+                            name,
+                            &payload,
+                            &args,
+                            is_private,
+                        ),
                     ));
                 } else {
                     match plugin {
                         Some(plugin_url) => {
                             // send to specific plugin(s)
                             pipe_to_specific_plugins(
                                 PipeSource::Keybind,
+                                Some(cli_client_id),
```

Comment:
I think this should be a `None`. Even though in this case cli_client_id is will always be None (since this is coming from a keybinding), we shouldn't re-use it in case this behavior somehow changes in the future. Then, if we want to add the cli_client_id we'll do it explicitly.